stats/opentelemetry/opentelemetry.go

```diff
@@ -96,6 +100,9 @@ type MetricsOptions struct {
 // configured for an individual metric turned on, the API call in this component
 // will create a default view for that metric.
 func DialOption(o Options) grpc.DialOption {
+	if o.MetricsOptions.Context == nil {
+		o.MetricsOptions.Context = context.Background()
+	}
```

Comment:
Shouldn't this be passed in through the first function parameter, and be mandatory, instead of through a struct?

Or maybe we should replace the places where we access this with a `context.TODO` since we don't really know how it should work?  Or even invent our own `emptyContext` or something that implements `context.Context` but is effectively the same thing to avoid linters, since maybe we're actually sure it's impossible to ever pass a context here?

Reply:
Discussed offline; decided context.TODO and ask OpenTelemetry team the technical reasons their records on instruments take a context (which is ignored) whereas no context in Java and C.

Reply:
"They are currently used to record exemplars (here: https://github.com/open-telemetry/opentelemetry-go/blob/99f830ba01c23f5b0cdc55357f1d5c82bb83e344/sdk/metric/internal/exemplar/storage.go#L64)."

"If you call Record() or Add() with the context returned by tracer.Start(), you will get exemplars from the spans you record on the metrics collected." - David, so since these don't apply to traces I think ignoring makes sense here.